apps/src/codebridge/FileBrowser/index.tsx

```diff
@@ -259,6 +291,18 @@ export const FileBrowser = React.memo(() => {
     [renameFolder, project.folders]
   );
 
+  const toggleFileVisibility: FilesComponentProps['toggleFileVisibility'] =
+    useMemo(
+      () => fileId => {
+        const file = project.files[fileId];
+        // file.visible could be undefined.
+        // We only want to show the file if it's currently explicitly hidden.
+        const visibility = file.visible === false;
```

Comment:
could we avoid some of these explicit checks by changing the field from `visible` to `hidden`, then we could just do checks on `file.hidden`, and if it's undefined or false we will show the file?

Reply:
I had this nagging thought about that while I was working, but I think my conscience felt I was in too deep. You're totally correct though and I'm going to rework this. "Hidden" also better aligns to how we've been describing this feature as well.